simulation_ws/src/deepracer_simulation/urdf/racecar.gazebo

```diff
@@ -63,28 +63,23 @@
 <xacro:unless value="$(optenv DISABLE_GAZEBO_CAMERA false)">
   <gazebo reference="camera_link">
     <sensor type="camera" name="zed_camera_left_sensor">
-      <update_rate>60.0</update_rate>
+      <update_rate>15.0</update_rate>
       <!-- math.atan(320 / 687.8065795898438) * 2 -->
       <camera name="zed_camera_left_camera">
-        <horizontal_fov>1.1931415</horizontal_fov>
+        <horizontal_fov>1.13</horizontal_fov>
         <image>
-          <width>160</width>
-          <height>120</height>
-          <format>B8G8R8</format>
+          <width>640</width>
+          <height>480</height>
+          <format>R8G8B8</format>
```

Comment:
We expect the format to be BGR in these 2 places that we might have to update: 
https://github.com/aws-robotics/aws-robomaker-sample-application-deepracer/blob/master/simulation_ws/src/sagemaker_rl_agent/markov/environments/deepracer_env.py#L205-L206
https://github.com/awslabs/amazon-sagemaker-examples/blob/master/reinforcement_learning/rl_deepracer_robomaker_coach_gazebo/src/robomaker/environments/deepracer_env.py#L206

Reply:
As a side note, do we need to change this format?

Reply:
Thanks for this catch. I do have a few updates to the SageMaker notebook files, and I'll file a PR soon for that. 

There is no need to change this format technically. We use RGB for the simulation application in DeepRacer production app, so it will be good to be consistent.